test/discr/discr_ops_test.py

```diff
@@ -0,0 +1,218 @@
+    res_op = odl.ResizingOperator(space, res_space)
+
+    assert res_op.domain == space
+    assert res_op.range == res_space
+    assert res_op.offset == (0, 5)
+    assert res_op.pad_mode == 'constant'
+    assert res_op.pad_const == 0.0
+    assert res_op.is_linear
+
+    # Implicit range via ran_shp and offset
+    res_op = odl.ResizingOperator(space, ran_shp=(20, 15), offset=[0, 5])
+    assert np.allclose(res_op.range.min_corner, res_space.min_corner)
+    assert np.allclose(res_op.range.max_corner, res_space.max_corner)
+    assert np.allclose(res_op.range.cell_sides, res_space.cell_sides)
+    assert res_op.range.dtype == res_space.dtype
+    assert res_op.offset == (0, 5)
+
+    # Different padding mode
+    res_op = odl.ResizingOperator(space, res_space, pad_const=1)
+    assert res_op.pad_const == 1.0
+    assert not res_op.is_linear
+
+    res_op = odl.ResizingOperator(space, res_space, pad_mode='symmetric')
+    assert res_op.pad_mode == 'symmetric'
+    assert res_op.is_linear
+
+
+def test_resizing_op_call(fn_impl):
+
+    # Minimal test since this operator only wraps resize_array
+    space = odl.uniform_discr([0, -1], [1, 1], (4, 5))
+    res_space = odl.uniform_discr([0, -0.6], [2, 0.2], (8, 2))
+    res_op = odl.ResizingOperator(space, res_space)
+    out = res_op(space.one())
+    true_res = np.zeros((8, 2))
+    true_res[:4, :] = 1
+    assert np.array_equal(out, true_res)
+
+    out = res_space.element()
+    res_op(space.one(), out=out)
+    assert np.array_equal(out, true_res)
+
+
+def test_resizing_op_deriv():
+
+    # Only non-trivial case is constant padding with const != 0
+    space = odl.uniform_discr([0, -1], [1, 1], (4, 5))
+    res_space = odl.uniform_discr([0, -0.6], [2, 0.2], (8, 2))
+    res_op = odl.ResizingOperator(space, res_space, pad_const=1.0)
+    res_op_deriv = res_op.derivative(space.one())
+    assert res_op_deriv.pad_mode == 'constant'
+    assert res_op_deriv.pad_const == 0.0
+
+
+pad_modes = [('constant', 0), ('constant', 1), 'symmetric', 'periodic',
+             'order0', 'order1']
+pad_mode_ids = [' constant=0 ', ' constant=1 ', ' symmetric ', ' periodic ',
+                ' order0 ', ' order1 ']
+
+
+@pytest.fixture(scope="module", params=pad_modes, ids=pad_mode_ids)
+def pad_mode(request):
+    return request.param
+
+
+def test_resizing_op_inverse(pad_mode, dtype, fn_impl):
+
+    if isinstance(pad_mode, tuple):
+        pad_mode, pad_const = pad_mode
+    else:
+        pad_const = 0.0
+
+    space = odl.uniform_discr([0, -1], [1, 1], (4, 5), dtype=dtype,
+                              impl=fn_impl)
+    res_space = odl.uniform_discr([0, -1.4], [1.5, 1.4], (6, 7), dtype=dtype,
+                                  impl=fn_impl)
+    res_op = odl.ResizingOperator(space, res_space, pad_mode=pad_mode,
+                                  pad_const=pad_const)
+
+    # Only left inverse if the operator extentds in all axes
+    x = space.element(np.arange(space.size).reshape(space.shape))
```

Comment:
Should simply use `odl.util.testutils.example_element(space)`, it would be more obvious what you mean.